.travis.yml

```diff
@@ -53,39 +53,39 @@ jobs:
       env:
         - AGP_VERSION=3.4.0
         - GRADLE_WRAPPER_VERSION=5.1.1
-      script: bundle exec maze-runner -c --verbose
+      script: bundle exec maze-runner -c --verbose --fail-fast
 
       # AGP 3.5.0 E2E tests
     - name: AGP 3.5.0 E2E tests
       env:
         - AGP_VERSION=3.5.0
         - GRADLE_WRAPPER_VERSION=5.4.1
-      script: bundle exec maze-runner -c --verbose
+      script: bundle exec maze-runner -c --verbose --fail-fast
 
       # AGP 3.6.0 E2E tests
     - name: AGP 3.6.0 E2E tests
       env:
         - AGP_VERSION=3.6.0
         - GRADLE_WRAPPER_VERSION=5.6.4
-      script: bundle exec maze-runner -c --verbose
+      script: bundle exec maze-runner -c --verbose --fail-fast
 
       # AGP 4.0.0 E2E tests
     - name: AGP 4.0.0 E2E tests
       env:
         - AGP_VERSION=4.0.0
         - GRADLE_WRAPPER_VERSION=6.1.1
-      script: bundle exec maze-runner -c --verbose
+      script: bundle exec maze-runner -c --verbose --fail-fast
 
     # AGP 4.1.0 E2E tests
     - name: AGP 4.1.0 E2E tests
       env:
         - AGP_VERSION=4.1.0-beta04
         - GRADLE_WRAPPER_VERSION=6.5.1
-      script: bundle exec maze-runner -c --verbose
+      script: bundle exec maze-runner -c --verbose --fail-fast
 
       # AGP 4.2.0 E2E tests
     - name: AGP 4.2.0 E2E tests
       env:
-        - AGP_VERSION=4.2.0-alpha07
-        - GRADLE_WRAPPER_VERSION=6.6-rc-6
-      script: bundle exec maze-runner -c --verbose
+        - AGP_VERSION=4.2.0-beta03
```

Comment:
If this was already running on Travis and was passing, does that mean we don't have tests for the changes in this PR?

Reply:
Good question - the breaking change was introduced in AGP 4.2.0-alpha14, - so alpha07 would pass CI fine using the old API.